apps/mobile/src/screens/get-location/get-location.screen.tsx

```diff
@@ -5,15 +5,16 @@ import { useEffect } from 'react';
 import { Alert, Linking } from 'react-native';
 
 import { ActionButtonsGroup } from './compose';
-import localizationImg from '../../../assets/localization.png';
-import SafeArea from '../../components/safe-area/safe-area';
-import AppStatusBar from '../../components/status-bar/status-bar.component';
-import { useAuth } from '../../hooks/use-auth-provider';
-import useLocale from '../../hooks/use-locale';
-import useGeolocation from '../../hooks/use-user-location';
-import { Warnings } from '../../hooks/use-user-location/use-user-location';
-import Routes from '../../routes';
-import theme from '../../theme';
+
+import localizationImg from '@/assets/localization.png';
+import SafeArea from '@/components/safe-area/safe-area';
+import AppStatusBar from '@/components/status-bar/status-bar.component';
+import { useAuth } from '@/hooks/use-auth-provider';
+import useLocale from '@/hooks/use-locale';
+import useGeolocation from '@/hooks/use-user-location';
+import { Warnings } from '@/hooks/use-user-location/use-user-location';
```

Comment:
Can you reexport `Warnings` from the index file so we can import both modules at the same line?